core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -169,6 +172,25 @@ oppia.controller('StatisticsTab', [
                 return false;
               };
 
+              var _getVisualizationsHtml = function() {
```

Comment:
This "manual assembly" is a bit ick. I wonder if there's a better way to do this, similar to what we do for interactions.